apps/web/components/team/screens/Team.tsx

```diff
@@ -12,6 +14,7 @@ type MemberType = MembersType[number];
 
 const Member = ({ member, teamName }: { member: MemberType; teamName: string | null }) => {
   const { t } = useLocale();
+  const parsedBio = parse(member.bio || "");
```

Comment:
Are we saving this data as HTML or Markdown? And if not markdown, why?

Reply:
It is saved as HTML. The `Editor` component creates a HTML with styles from the` stylesEditor.css` file.  Should I convert the bio to markdown before saving it?